dpnp/backend/include/dpnp_gen_2arg_3type_tbl.hpp

```diff
@@ -35,10 +35,50 @@
  *
  */
 
+#include "dpnp_async.hpp"
+
+#ifndef MACRO_2ARG_3TYPES_OP_ASYNC
```

Comment:
Better to move the macros calls to separate header `dpnp_gen_2arg_3type_async_tbl.hpp`.
Also I would remove the macros to `MACRO_2ARG_3TYPES_ASYNC_OP`.